fe/src/components/Users/Users.re

```diff
@@ -34,13 +34,11 @@ let prepareTeamIcons = teamName => {
   <span className=icon />;
 };
 
-let getTrend = lastSummaryRatingDiff =>
-  if (lastSummaryRatingDiff < 0) {
-    DOWN;
-  } else if (lastSummaryRatingDiff == 0) {
-    STAY;
-  } else {
-    UP;
+let getTrend = trendRatingDiff =>
+  switch (trendRatingDiff) {
+  | ratingDiff when ratingDiff < 0 => DOWN
```

Comment:
Jak to działa ;o? Dziwnie to wygląda.

Reply:
Pattern matching ma taką opcję:
https://reasonml.github.io/docs/en/pattern-matching

Generalnie tak jak pisałem wcześniej: switch domyślenie umie tylko porównywać (ewentualnie robić jednocześnie destrukcję i ją porównywać). Aby robić sprawdzanie jakiś logicznych warunków (jak tutaj sprawdzanie przedziałów) trzeba używać słówka when. Jak coś w tym linku co przesłałem ładnie jest wszystko wyjaśnione